server/channels/api4/emoji_test.go

```diff
@@ -430,6 +430,7 @@ func TestDeleteEmoji(t *testing.T) {
 	th.AddPermissionToRole(model.PermissionDeleteOthersEmojis.Id, model.SystemUserRoleId)
 
 	client.Logout(context.Background())
```

Comment:
You need to assign the return value of `client.Logout` to a variable called `err` and then check it.

Reply:
@TheInvincibleRalph Can you please push your changes? Thanks!

Reply:
Done. Thank you!